### DISCUSSION TITLE
Zotero Connector 新增条目时增加冲突判断功能

### DISCUSSION THREAD

**Discussion by Maxpicca-Li** [zotero-chinese/wiki#311] [好点子 / Ideas]
在“我的文库”中，我们往往会发现很多重复的条目。希望可以在 Zotero Connector 新增条目时，增加冲突判断功能：
1. 如果是新条目，直接添加
2. 如果已有数据库中存在同名词条，则询问是否新增附件/附件链接

**Comment by northword** [comment]
追踪 https://github.com/northword/zotero-format-metadata/issues/74 中提到的论坛帖子和 issue。
